ibis/impala/ddl.py

```diff
@@ -761,38 +313,14 @@ def compile(self):
         return ' '.join([create_decl, impala_sig]) + ' ' + '\n'.join(tokens)
 
 
-class DropFunction(DropObject):
-    def __init__(
-        self, name, inputs, must_exist=True, aggregate=False, database=None
-    ):
-        super().__init__(must_exist=must_exist)
-        self.name = name
-        self.inputs = tuple(map(dt.dtype, inputs))
-        self.must_exist = must_exist
-        self.aggregate = aggregate
-        self.database = database
-
+class ImpalaDropFunction(DropFunction):
```

Comment:
is there a reason you renamed this? shouldn't it just be DropFunction? (and change the import to avoid the name conflict)

Reply:
there was a function with impala in the name so i figured that was a specific implementation for impala.  so i took that part out for the `base_sql` implementation and made this `ImpalaDropFunction` with that function added.

do you think that this function isnt specific to impala and can just be moved to `DropFunction` in `base_sql`?

Reply:
i think the location is fine, just puzzled why you added the Impala prefix; we don't do this anywhere else; i would just rename it to DropFunction and call it a day.

Reply:
Just updated and CI is green. Let me know if anything else.